packages/vite/src/node/plugins/esbuild.ts

```diff
@@ -403,14 +403,17 @@ const tsconfckParseOptions: TSConfckParseOptions = {
 }
 
 async function initTSConfck(config: ResolvedConfig) {
-  tsconfckParseOptions.cache!.clear()
   const workspaceRoot = searchForWorkspaceRoot(config.root)
+  debug(`init TSConfck (root: ${colors.cyan(workspaceRoot)})`)
+
+  tsconfckParseOptions.cache!.clear()
   tsconfckParseOptions.root = workspaceRoot
   tsconfckParseOptions.tsConfigPaths = new Set([
     ...(await findAll(workspaceRoot, {
       skip: (dir) => dir === 'node_modules' || dir === '.git'
     }))
   ])
+  debug(`init TSConfck end`)
```

Comment:
```suggestion
  debug(`init tsconfck (root: ${colors.cyan(workspaceRoot)})`)

  tsconfckParseOptions.cache!.clear()
  tsconfckParseOptions.root = workspaceRoot
  tsconfckParseOptions.tsConfigPaths = new Set([
    ...(await findAll(workspaceRoot, {
      skip: (dir) => dir === 'node_modules' || dir === '.git'
    }))
  ])
  debug(`init tsconfck end`)
```

@dominikg suggested that `tsconfck` should be all lowercase 😬

Reply:
Oh, I thought it as `TSConfck` because of function/type names (`TSConfckParseError`, `initTSConfck`, ...).
Updated 👍